tests/integration/auth/runner.py

```diff
@@ -241,70 +200,126 @@ def cleanup():
 
     # Run the test with all combinations of permissions
     print("\033[1;36m~~ Starting query test ~~\033[0m")
+    for db in ["memgraph", "db1"]:
+        print("\033[1;36m~~ Running against db {} ~~\033[0m".format(db))
+        execute_user_queries(["USE DATABASE {}".format(db)], should_fail=True, failure_message=UNAUTHORIZED_ERROR)
```

Comment:
Why `USE DATABASE memgraph` should fail? It's a default database

Reply:
Because we revoked its access from the user.
"memgraph" is THE default database, but each user can have a different private default database.